README.md

```diff
@@ -125,6 +125,7 @@ There are many flags that you can modify as you see fit.
 | `SHOW_TOTAL`       | `false`, `true`                                                         | Show total coding time                                                            |
 | `SHOW_MASKED_TIME` | `false`, `true`                                                         | Adds total coding time including unclassified languages (overrides: `SHOW_TOTAL`) |
 | `STOP_AT_OTHER`    | `false`, `true`                                                         | Stop when language marked as `Other` is retrieved (overrides: `LANG_COUNT`)       |
+| `IGNORE_LANGUAGES` | `binary,json`                                                           | Hide any language you wouldn't like to show in your stats                         |
```

Comment:
The default value of `IGNORED_LANGUAGES` is an empty string <code> </code>, not `binary` and they are space (not coma) separated.

```diff
- | `IGNORE_LANGUAGES` | `binary,json` | Hide any language you wouldn't like to show in your stats |
+ | `IGNORED_LANGUAGES` | <code> </code>, `Binary YAML JSON TOML`                                 | Hide languages from your stats                                                    |
```

> **Note**:
> It's `IGNORED_LANGUAGES` not `IGNORE_LANGUAGES`
>
> **Preview**:
>
> | Environment flag    | Options (`Default`, `Other`, ...)                                       | Description                                                                       > |
> | ------------------- | ----------------------------------------------------------------------- | --------------------------------------------------------------------------------- |
> | `IGNORED_LANGUAGES` | <code> </code>, `Binary YAML JSON TOML`                                 | Hide languages from your stats                                                    |